messaging-activemq/src/main/resources/schema/wildfly-messaging-activemq_1_1.xsd

```diff
@@ -68,6 +68,7 @@
                 <xs:complexType>
                     <xs:attribute name="enabled" type="xs:boolean" />
                     <xs:attribute name="domain" type="xs:string" />
+                    <xs:attribute name="elytron-domain" type="xs:string"/>
```

Comment:
I noticed that the parser order does not match the XSD. Could you update the parser code to match the XSD attribute order, please?